user-contrib/Ltac2/tac2quote.mli

```diff
@@ -32,7 +32,7 @@ val of_variable : Id.t CAst.t -> raw_tacexpr
 
 val of_ident : Id.t CAst.t -> raw_tacexpr
 
-val of_constr : Constrexpr.constr_expr -> raw_tacexpr
+val of_constr : Id.t list -> Constrexpr.constr_expr -> raw_tacexpr
```

Comment:
Nitpicking, but this could be an optional argument called `scopes` so that it reduces the unrelated changes.

Reply:
Sounds like a good idea. I’m unsure about the “scopes” name: it is unclear whether it refers to ltac2 scopes or Coq notation scopes.

What about “delimiters”?

Reply:
Fine.

Reply:
Done.